stable/distribution/templates/redis-connection-secret.yaml

```diff
@@ -1,17 +0,0 @@
-apiVersion: v1
```

Comment:
Why has this been deleted? Where would the password be stored in a case where the user didn't provide an existing secret?

Reply:
I moved this logic to `distribution-secrets.yaml` file